docs/deployment/kubernetes.md

```diff
@@ -166,14 +170,14 @@ The UI is exposed on NodePort 31112.
 
 Click "New Function" and fill it out with the following:
 
-| Field      | Value                        |
--------------|------------------------------|
-| Service    | nodeinfo                     |
-| Image      | functions/nodeinfo:latest    |
-| fProcess   | node main.js                 |
-| Network    | default                      |
+| Field    | Value                     |
+| -------- | ------------------------- |
+| Service  | nodeinfo                  |
+| Image    | functions/nodeinfo:latest |
+| fProcess | node main.js              |
+| Network  | default                   |
 
-* Test the function
+- Test the function
```

Comment:
Not sure why this was changed? And if we're going to change the style that should be done in the whole site rather than making one file inconsistent.